proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -68,6 +68,8 @@ message ListRepositoryCommitsByBranchRequest {
   uint32 page_size = 4;
   string page_token = 5;
   bool reverse = 6;
+  // Allow the server to omit tags from the response.
```

Comment:
It feels like we'd want a general-purpose solution to this across our API calls, as opposed to a one-off. Is there a performance issue we're having that necessitates this?